client/blocks/login/login-header.tsx

```diff
@@ -173,25 +171,18 @@ export function getHeaderText(
 		if ( isBlazeProOAuth2Client( oauth2Client ) ) {
 			headerText = translate( 'Log in to your Blaze Pro account' );
 		}
-	}
-
-	if ( isWooJPC ) {
+	} else if ( isWooJPC ) {
 		const isLostPasswordFlow = currentQuery.lostpassword_flow === 'true';
-		switch ( true ) {
-			case isLostPasswordFlow:
-				headerText = translate( "You've got mail" );
-			case twoFactorEnabled:
-				headerText = translate( 'Authenticate your login' );
-			default:
-				headerText = translate( 'Log in to your account' );
+		if ( isLostPasswordFlow ) {
+			headerText = translate( "You've got mail" );
+		} else if ( twoFactorEnabled ) {
+			headerText = translate( 'Authenticate your login' );
+		} else {
+			headerText = translate( 'Log in to your account' );
 		}
-	}
-
-	if ( isFromMigrationPlugin ) {
+	} else if ( isFromMigrationPlugin ) {
```

Comment:
I'm curious about this "migration plugin" flow. It's active when there is a `from=wpcom-migration` query param, and its layout is now also broken.

@jeherve @anomiex Is this `wpcom-migration` plugin and the redirects to `/log-in?from=wpcom-migration` (or Jetpack Connect with the same query param) still active? I see that the `Automattic/wpcom-migration` repo has been archived a few months ago, the "Move to WordPress.com" plugin is no longer in the .org repo, and there is only a "Migrate to WordPress.com" plugin based on BlogVault.

That sounds almost like this flow is no longer used and could be removed.

Reply:
The version of the plugin that connected to WordPress.com via this flow is indeed no longer in use. The new version of the plugin doesn't use that flow. I think we could remove it.

Reply:
All I know about the `wpcom-migration` is that it was created in the monorepo, had a few releases, then nothing for a while until apparently they decided to replace it with a reskin of an existing (external) plugin. Whether this code path is still used from somewhere (e.g. the existing plugin from the external partner) I have no idea.

Reply:
Sounds like a good candidate for a follow-up PR!